rebasehelper/cli.py

```diff
@@ -49,12 +49,51 @@ def _format_action_invocation(self, action):
         return text.replace(' BUILDER_OPTIONS', '=BUILDER_OPTIONS')
 
     def _expand_help(self, action):
+        if hasattr(action, "actual_default"):
+            action.default = action.actual_default
         if isinstance(action.default, list):
             default_str = ','.join([str(c) for c in action.default])
             action.default = default_str
         return super(CustomHelpFormatter, self)._expand_help(action)
 
 
+class CustomAction(argparse.Action):
+    def __init__(self, option_strings,
+                 switch=False,
+                 actual_default=None,
+                 dest=None,
+                 default=None,
+                 nargs=None,
+                 required=False,
+                 type=None,
+                 metavar=None,
+                 help=None,
+                 choices=None):
+
+        super(CustomAction, self).__init__(
+            option_strings=option_strings,
+            dest=dest,
+            default=default,
+            required=required,
+            metavar=metavar,
+            type=type,
+            help=help,
+            choices=choices)
+
+        self.switch = switch
+        if self.switch:
+            self.nargs = 0
+        else:
+            self.nargs = nargs
+        self.actual_default = actual_default
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        if self.switch:
+            setattr(namespace, self.dest, True)
+        else:
+            setattr(namespace, self.dest, values)
```

Comment:
Again, this could be written also as:
```python
setattr(namespace, self.dest, values if not self.switch else True)
```
But it's a matter of preference, if you think your way is more readable, keep it that way.